Packs/rasterize/Integrations/rasterize/rasterize_test.py

```diff
@@ -317,3 +317,9 @@ def test_log_warning():
     from rasterize import pypdf_logger
     assert pypdf_logger.level == logging.ERROR
     assert pypdf_logger.level == logging.ERROR
+
+
+def test_poppler_version():
+    import pdf2image
+    poppler_version = pdf2image.pdf2image._get_poppler_version("pdftoppm")
```

Comment:
lets have an e2e test like we have in ParseEmailFiles where we use test files and run the command and check for the output

Reply:
Poppler is a C executable, and pdf2image is a Python wrapper.
In order to reproduce the issue I'll need a DocuSign doc, which we don't have.